engines/experiment/manager/types.go

```diff
@@ -23,6 +24,33 @@ type Engine struct {
 	HomePageURL string `json:"home_page_url"`
 }
 
+type RemoteUI struct {
+	// Name is the name of the remote app declared in the Module Federation plugin
+	Name string `json:"name"`
+	// URL is the Host + Remote Entry file at which the remote UI can be found
+	URL string `json:"url"`
+}
+
+type CustomExperimentManagerConfig struct {
+	// RemoteUI specifies the information for the custom experiment engine UI to be
```

Comment:
Is a UI implementation compulsory for `CustomExperimentManager`, or do we just expect the screenshot you've attached in the PR description?

Reply:
The screenshot in the PR description is for the standard experiment manager. It will only work for standard ones.

> Is a UI implementation compulsory for CustomExperimentManager

Yes. More info here: `engines/experiment/docs/developer_guide.md`